spark/v3.4/spark-extensions/src/jmh/java/org/apache/iceberg/spark/PlanningBenchmark.java

```diff
@@ -215,12 +231,21 @@ private DeleteFile loadAddedDeleteFile() {
   }
 
   private void initDataAndDeletes() throws NoSuchTableException {
+    Schema schema = table.schema();
+    PartitionSpec spec = table.spec();
+    LocationProvider locations = table.locationProvider();
+
     for (int partitionOrdinal = 0; partitionOrdinal < NUM_PARTITIONS; partitionOrdinal++) {
       Dataset<Row> inputDF =
-          randomDataDF(table.schema(), NUM_ROWS_PER_DATA_FILE)
+          randomDataDF(schema, NUM_ROWS_PER_DATA_FILE)
               .drop(PARTITION_COLUMN)
-              .withColumn(PARTITION_COLUMN, lit(partitionOrdinal));
-      appendAsFile(inputDF);
+              .withColumn(PARTITION_COLUMN, lit(partitionOrdinal))
+              .drop(SORT_KEY_COLUMN)
+              .withColumn(SORT_KEY_COLUMN, lit(Integer.MIN_VALUE));
```

Comment:
Setting `SORT_KEY_COLUMN` to `Integer.MIN_VALUE` so that these files won't actually match the sort key predicate. Previously, all files matched the sort key predicate.